src/sass/messaging/partials/_message-compose.scss

```diff
@@ -48,8 +48,9 @@
 .msg-subject-group {
   margin-top: 0;
   padding: 1.5rem 0;
-
+  
   @media (min-width: $medium-screen) {
+    border-left: 4px solid orange;
```

Comment:
Are the fields' left borders supposed to have color by default (when there's no error)? If so, this seems to remain orange even when I pick 'Other' for the category and leave the subject empty.